backend/conftest.py

```
@@ -0,0 +1,174 @@
+    return mock
+
+
+# ---------------------------------------------------------------------------
+# Fixtures
+# ---------------------------------------------------------------------------
+
+@pytest.fixture(autouse=True)
+def _patch_heavy_imports(monkeypatch):
+    """Monkey-patch heavy third-party imports before server.py is loaded."""
+    # Prevent real NLTK downloads
+    monkeypatch.setattr("nltk.download", lambda *a, **kw: None)
```

Ruff (0.15.0)

[warning] Unused lambda argument: `a`

(ARG005)